extensions/wikia/DesignSystem/i18n/en.json

```diff
@@ -73,5 +74,22 @@
   "global-navigation-wikis-community-central": "Community Central",
   "global-navigation-wikis-explore": "Explore Wikis",
   "global-navigation-wikis-fandom-university": "Fandom University",
-  "global-navigation-wikis-header": "Wikis"
+  "global-navigation-wikis-header": "Wikis",
+  "notifications-no-notifications-message": "No notifications yet.",
+  "notifications-mark-all-as-read": "Mark all as read",
+  "notifications-replied-by-multiple-users-with-title": "__mostRecentUser__ and __number__ other users <b>replied</b> to __postTitle__",
```

Comment:
Please use https://github.com/Wikia/app/blob/dev/extensions/wikia/DesignSystem/i18n/build.js to convert variables to format supported by MW.